modules/claims_api/app/controllers/claims_api/v1/forms/power_of_attorney_controller.rb

```diff
@@ -120,6 +121,10 @@ def validate
 
         private
 
+        def enable_vmbs_access?
```

Comment:
is this supposed to be called on line 50 above? i dont see it being called anywhere in this file currently.

Reply:
oof yeah I accidentally had stuff in v0 at first and must have forgotten it moving things over